packages/components/src/InputFile/InputFile.tsx

```diff
@@ -358,14 +369,14 @@ function getLabels(
 ) {
   let buttonLabel = multiple ? "Upload Files" : "Upload File";
   let hintText = multiple
-    ? "or drag files here to upload"
-    : "or drag a file here to upload";
+    ? "Select or drag files here to upload"
```

Comment:
I've experimented with a few different ways of writing this function now, since there's so much repetition, but I keep ending up with an equally convoluted, hard to read function.  Any feedback/insight on this would be appreciated!

Reply:
I was able to rewrite it as
```ts
function getLabels(
  providedButtonLabel: string | undefined,
  multiple: boolean,
  allowedTypes: string | string[],
) {
  const fileType =
    allowedTypes === "images" || allowedTypes === "basicImages"
      ? "Image"
      : "File";
  let buttonLabel = multiple ? `Upload ${fileType}s` : `Upload ${fileType}`;
  const fileTypeDeterminer = fileType === "Image" ? "an" : "a";
  const hintText = multiple
    ? `Select or drag ${fileType.toLowerCase()}s here to upload`
    : `Select or drag ${fileTypeDeterminer} ${fileType.toLowerCase()} here to upload`;

  if (providedButtonLabel) buttonLabel = providedButtonLabel;

  return { buttonLabel, hintText };
}
```
This didn't change any snapshots, I feel like it is more readable but that may not be the case

Reply:
Okay that's verrry close to what I had - in some ways I feel like it's a lot but it's still readable, gets the job done and dry's it up. Ok I'll update that

Reply:
Updated here 1d39876ac162b2d64ba3a01338a49e7843c8d5e3
Also just changed one InputFile in Variations and Sizes example to be plural and better fit the description I've set.